internal/build/docker_builder.go

```diff
@@ -206,6 +206,27 @@ func (d *DockerBuilder) BuildImage(ctx context.Context, ps *PipelineState, refs
 func (d *DockerBuilder) buildToDigest(ctx context.Context, spec v1alpha1.DockerImageSpec, filter model.PathMatcher, allowBuildkit bool) (digest.Digest, []v1alpha1.DockerImageStageStatus, error) {
 	var contextReader io.Reader
 
+	buildContext := spec.Context
+
+	// Treat context: "-" as an empty context.
+	if buildContext == "-" {
+		emptyContextDir, err := os.MkdirTemp("", "tilt-dockercontext-")
```

Comment:
Is there no direct way to tell this to the builder? 🙈

Reply:
i think there are ways to do it but they're fairly complex to invoke? since the api is more optimized for the case where you're uploading a tarball.